master/buildbot/status/web/hooks/github.py

```diff
@@ -100,10 +101,10 @@ def handle_push(self, payload):
         user = None
         # user = payload['pusher']['name']
         repo = payload['repository']['name']
-        repo_url = payload['repository']['url']
+        repo_url = payload['repository']['clone_url']
         # NOTE: what would be a reasonable value for project?
         # project = request.args.get('project', [''])[0]
-        project = payload['repository']['full_name']
+        project = payload['repository']['name']
```

Comment:
It seems like a distinct change from the rest of the PR, and one that could cause compatibility problems.